### PR TITLE
Add default background white for the body tag

### PR DESCRIPTION
**Pull request by stephane-**
Hello,
On some old browsers the background appears black. The site is therefore not very usable. The fix does not change anything in modern browsers but fix the background color on some old browsers.
I do not submit my compiled css because I do not have the same configuration, and my compiler create lot of change on compiled file.

### CODE REVIEW

**Comment by fguillot** [comment]
Which old browsers are you talking about?

**Comment by stephane-** [comment]
Hello,
You can test it with connected TV (No smart) and more easily with steam browser (overlay or big picture).

Example : 
blackkanboard

File: assets/sass/_reset.sass [hunk 1]
```diff
@@ -22,6 +22,7 @@ body
   color: color('primary')
   font-family: $text-font
   text-rendering: optimizeLegibility
+  background-color: color('white')
```
Comment (fguillot) [inline assets/sass/_reset.sass]: It should be `color('default')` according to the change you made in `_variables.sass`?

**Comment by stephane-** [comment]
Yes, sorry for that. I change the var name when I clean the code

**Comment by fguillot** [comment]
Your commits have been merged into master.

**Pull request closed without merging**